static/src/javascripts/projects/common/modules/commercial/dfp.js

```diff
@@ -292,7 +292,7 @@ define([
             window.googletag.cmd.push(defineSlots);
 
             // We do not want lazy loading on pageskins because it messes up the roadblock
-            if (config.switches.viewability && !config.page.hasPageSkin) {
+            if (config.switches.viewability && (!config.page.hasPageSkin || (config.page.hasPageSkin && detect.getBreakpoint() !== 'wide'))) {
```

Comment:
Isn't this the same as `... && (!(config.page.hasPageSkin && detect.getBreakpoint() === 'wide')))` ?